crates/fluvio-cluster/src/render.rs

```diff
@@ -1,4 +1,48 @@
+use indicatif::ProgressBar;
+
 pub trait ProgressRenderedText {
     /// Rendered text of the last finished step in the progress
     fn msg(&self) -> String;
 }
+
+#[derive(Debug)]
+pub enum ProgressRenderer {
+    /// Render the progress using eprintln macro
+    Std,
+    /// Render the progress using Indicatiff
+    Indicatiff(ProgressBar),
+}
+
+impl ProgressRenderer {
+    pub fn println(&self, msg: String) {
+        match self {
+            ProgressRenderer::Std => eprintln!("{}", msg),
+            ProgressRenderer::Indicatiff(pb) => pb.println(&msg),
+        }
+    }
+
+    pub fn set_message(&self, msg: String) {
```

Comment:
same here. `set_message(&self, msg: &str)` would be better

Reply:
I was having issues with the fact that set_message in progress bar has as parameter a  `impl Into<Cow<'static, str>>);`.  I will change this to also use this